idaes/gas_solid_contactors/properties/oxygen_iron_OC_oxidation/gas_phase_thermo.py

```diff
@@ -576,18 +581,21 @@ def _diffusion_comp(self):
         # Component diffusion in a gas mixture - units of cm2/s to help scaling
         self.diffusion_comp = Var(self._params.component_list,
                                   domain=Reals,
-                                  initialize=1e-5,
+                                  initialize=1.0,
                                   doc='Component diffusion in a gas mixture'
                                   '[cm2/s]',
                                   units=pyunits.cm**2/pyunits.s)
 
         def D_bin(i, j):
             # 1e3 used to multiply MW to convert from kg/mol to kg/kmol
+            # TODO: Does this factor of 1.43e-3 have units?
```

Comment:
The equation is empirical. I've tried to work out the units and it looks nasty. Something like **kg**^2.5 **s**^-5 **K**^-1.75 **kgmol**^-0.5